src/core/registration.c

```diff
@@ -291,6 +305,17 @@ QuicRegistrationTraceRundown(
     }
 
     CxPlatDispatchLockRelease(&Registration->ConnectionLock);
+
+    CxPlatLockAcquire(&Registration->ListenerLock);
+
+    for (CXPLAT_LIST_ENTRY* Link = Registration->Listeners.Flink;
+        Link != &Registration->Listeners;
+        Link = Link->Flink) {
+        QuicListenerTraceRundown(
+            CXPLAT_CONTAINING_RECORD(Link, QUIC_LISTENER, RegistrationLink));
+    }
+
+    CxPlatLockRelease(&Registration->ListenerLock);
```

Comment:
This doesn't need to be called from here, because it is already called from the Binding (which is called from the library).

Reply:
So it is unnecessary to trace the listener removed from registration.

ok.